random_joke.py

```diff
@@ -23,9 +23,23 @@ def multiple_joke():
         request_data = MultipleJokesRequestParams(**raw_request)
     except ValidationError as exc:
         return(str(exc))
-        
+
     return JokeApi.multiple_jokes(request_data.count)
 
+# mock/test translator
+@app.route("/multi-language-jokes")
+def test_translator():
```

Comment:
Is this supposed to be called `test_translator`?

Reply:
No, It all mine :D

Reply:
renamed